.github/workflows/docs.yaml

```diff
@@ -0,0 +1,67 @@
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "!**-rc"
```

Comment:
Did you try the following tags on your fork?
* `0.8.0-rc1`
* `0.8.0-rc2`
* `0.8.0`

Reply:
I've improved the glob!

Instead of dealing with negations (i.e. `all except `-rcN`) I'd suggest to use tags named as `release/x.y.z` or `rel/x.y.z`. AFAIR such tags are automatically protected by the default security rules at Apache Git and once created no one can modify them. WDYT ?